lib/api/core/httpRouter/routeHandler.js

```diff
@@ -34,30 +34,32 @@ const
  * @class RouteHandler
  * @param {string} url - parsed URL
  * @param {string} requestId
- * @param {object} query - HTTP request query parameters
```

Comment:
I'm not sure but I think you made a mistake. You have delete the wrong docstring param. You let requestId in docstring and remove query. I think you wanted do the opposite. :slightly_smiling_face: